README.md

```diff
@@ -227,6 +245,63 @@ AwsOpenTelemetryRumBuilder.create(config: config)?
 - [Read more about configuring the SDK with authentication](docs/auth.md)
 - See the [AwsOpenTelemetryAuth README](Sources/AwsOpenTelemetryAuth/README.md) for detailed explanations on AwsOpenTelemetryAuth
 
+## Crash Report Symbolication
+
+| Method          | Support Status   | Description                                                                                                                                                                                                                                                     |
+| --------------- | ---------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Server-side** | ❌ Not supported | Please [open an issue](https://github.com/aws-observability/aws-otel-swift/issues) to prioritize this important feature!                                                                                                                                        |
+| **On-device**   | ✅ Supported     | Enable [on-device symbolication via KSCrash](https://github.com/kstenerud/KSCrash?tab=readme-ov-file#enabling-on-device-symbolication) by setting Strip Style to Debugging Symbols, which comes with the tradeoff of approximately ~5% increase in binary size. |
+| **Offline**     | ✅ Supported     | We've provided [a script in `Examples/Symbolication`](Examples/Symbolication/) to help with offline symbolication via `atos`, which also includes swift demangling. Offline symbolication is always possible via XCode or the native `atos` tool .              |
+
+## Custom Events
```

Comment:
question: Should we document size constraints?

Reply:
oh yes, 30 kb attribute limit should be documented in custom events section